typescript/test/test_application_js/integration-test.js

```diff
@@ -0,0 +1,57 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const util = require('util');
+const child_process = require('child_process');
+const exec = util.promisify(child_process.exec);
+const fs = require('fs-extra');
+const path = require('path');
+const SHOWCASE_SERVER = path.join(__dirname, 'showcase-server');
+const serverProcess = require( './server');
+const GAPIC_SHOWCASE_VERSION = '0.5.0';
+const OS = 'uname';
+const ARCH='amd64';
+
+describe('IntegrationTest for showcase library', () => {
+  describe('Run integration test for generated showcase library', async function () {
+    it('download the server', async function () {
+      this.timeout(120000);
+      if (!fs.existsSync(SHOWCASE_SERVER)) {
+          fs.mkdirSync(SHOWCASE_SERVER);
+      }
+      // Download server
+      process.chdir(SHOWCASE_SERVER);
+      console.warn('enter server folder: ' + process.cwd());
+      try {
+        const command = 'curl -L https://github.com/googleapis/gapic-showcase/releases/download/v' + GAPIC_SHOWCASE_VERSION + '/gapic-showcase-' + GAPIC_SHOWCASE_VERSION + '-' + OS + '-' +ARCH +'.tar.gz > gapic-showcase-server.tar.gz';
+        await exec(command);
+      } catch (err) {
+        console.log('exec error:', err);
+      }
+    });
+    it('run the server and test', async function () {
+      serverProcess.run();
+      // Run test
+      try{
+          await exec('mocha index.js');
+        }catch (err) {
```

Comment:
formatting. Would be great to make `prettier` work for test `js` files in the same way as `gts` works for `ts` files (you can see an example in JS libraries).